src/features/canvass/utils/sortMetrics.ts

```diff
@@ -0,0 +1,11 @@
+import { ZetkinMetric } from 'features/areaAssignments/types';
+
+export default function sortMetrics(
+  metricsList: ZetkinMetric[]
+): ZetkinMetric[] {
+  return metricsList
+    ?.slice()
```

Comment:
What is the purpose of the question mark here? Per the argument type `ZetkinMetric[]`, this can never be undefined, so I don't think the question mark is needed.